src/pages/forgot-password/index.jsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import { useAuth } from "../../context/auth";
+import { Link } from "react-router-dom";
+
+export default function ForgotPassword() {
+   const { username } = useAuth();
+   const navigate = useNavigate();
+
+   useEffect(() => {
+      if (username) {
+         navigate("/dashboard");
+      }
+   }, [username]);
+
+   if (username) {
+      return null;
+   }
+
+   return (
+      <section className="bg-gray-50 dark:bg-gray-900">
+         <Link
+            to="/"
+            className="bg-[#D71E28] hover:bg-red-500 text-white font-bold py-2 px-4 rounded-full absolute top-6 left-6"
+         >
+            Back
+         </Link>
+         <div className="flex flex-col items-center justify-center px-6 py-8 mx-auto md:h-screen lg:py-0">
+            <div className="flex items-center justify-center text-5xl font-bold text-[#D71E28] cursor-default pb-2">
+               {" "}
+               ACU
+            </div>
+            <div className="w-full p-6 bg-white rounded-lg shadow dark:border md:mt-0 sm:max-w-md dark:bg-gray-800 dark:border-gray-700 sm:p-8">
+               <h2 className="mb-1 text-xl font-bold leading-tight tracking-tight text-gray-900 md:text-2xl dark:text-white">
+                  Forgot Password
+               </h2>
+               <form className="mt-4 space-y-4 lg:mt-5 md:space-y-5" action="#">
+                  <div>
+                     <label for="email" className="block mb-2 text-sm font-medium text-gray-900 dark:text-white">
+                        Enter your email
+                     </label>
+                     <input
+                        type="email"
+                        name="email"
+                        id="email"
+                        className="bg-gray-50 border border-gray-300 text-gray-900 sm:text-sm rounded-lg focus:ring-primary-600 focus:border-primary-600 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
```

Comment:
We aren't making a dark mode compatible website. You need to remove `dark:` class names.